lib/ui/pages/news_list/news_list_item.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:allthenews/domain/news.dart';
+import 'package:flutter/material.dart';
+
+class NewsListItem extends StatelessWidget {
+  final News news;
+
+  NewsListItem({Key key, @required this.news})
+      : assert(news != null),
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: <Widget>[
+        _buildImage(),
+        SizedBox(width: 20),
+        Expanded(
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              _buildTitle(context),
+              SizedBox(height: 8),
+              _buildSubtitle(context),
+            ],
+          ),
+        )
+      ],
+    );
+  }
+
+  Widget _buildTitle(BuildContext context) {
+    return Text(
+      news.title,
+      style: _getTitleStyle(context),
```

Comment:
wszystkie style wyniósłbym do osobnej klasy constants, ale to musielibyśmy ustalić jaką konwencję przyjmujemy, może są jakieś best practices

Reply:
title ma też inny font na designach